package.json

```diff
@@ -55,7 +55,8 @@
     "docs": "node scripts/generate-readme.js",
     "simple-integration-test": "jest --maxWorkers=5 simple-suite",
     "complex-integration-test": "jest --maxWorkers=5 integration",
-    "postinstall": "node ./scripts/postinstall.js"
+    "postinstall": "node ./scripts/postinstall.js",
+    "pre-release": "node ./scripts/pre-release.js"
```

Comment:
I would change this to `prepublish` or `prepack` so that it's always executed when we publish manually.

Otherwise we might miss it / keep this step in the release checklist and always need to remember it.